src/agora/txrx/txrx.cc

```diff
@@ -90,13 +93,19 @@ bool PacketTXRX::StartTxRx(Table<char>& buffer, Table<int>& buffer_status,
 }
 
 void PacketTXRX::SendBeacon(int tid, size_t frame_id) {
+  static double send_time = 0;
+  double time_now = GetTime::GetTimeUs() / 1000;
   size_t radio_lo = tid * cfg_->NumRadios() / socket_thread_num_;
   size_t radio_hi = (tid + 1) * cfg_->NumRadios() / socket_thread_num_;
 
   // Send a beacon packet in the downlink to trigger user pilot
   std::vector<uint8_t> udp_pkt_buf(cfg_->PacketLength(), 0);
   auto* pkt = reinterpret_cast<Packet*>(&udp_pkt_buf[0]);
 
+  std::printf("TXRX [%d]: Sending beacon for frame %zu tx delta %f ms\n", tid,
```

Comment:
Maybe it is better to keep this as a debug message and only print it when needed.